.rubocop.yml

```diff
@@ -0,0 +1,214 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Deciding on rubocop style should be in a separate PR from adding a new API class, particularly because a lot of unrelated code to adding VPC was changed because of these rubocop rules

Reply:
The rubocop_todo.yml is output from the --auto-gen-config. This file whitelists all of the exceptions found by rubocop so that you can start clean and slowly work through the offences when you have time. If you have your IDE set to lint having all of these previous errors can be overwhelming. This clears your technical debt and ensures that you are not adding new violations making IDE listing more useful.